### PR TITLE
Add SPDX as source of license attribute content

### PR DESCRIPTION
**Pull request by prince-chrismc**
Docs!

❓ This document is "prefered but not required" this might not be the best document. It does not seem to fit the "how to add packages" doc.

🅰️ https://github.com/conan-io/conan-center-index/pull/4584#discussion_r576494959

### CODE REVIEW

File: docs/reviewing.md [hunk 1]
```diff
@@ -51,6 +53,10 @@ the order above resembles the execution order of methods on CI. therefore, for i
 The contents of `test_package.c` or `test_package.cpp` should be as minimal as possible, including a few headers at most with simple instatiation of objects to ensure linkage
 and dependencies are correct.
 
+## License Attribute
+
+The manadatory license attribute of each recipe must **only** of [SPDX license](https://spdx.org/licenses/) [short Identifiers](https://spdx.dev/ids/)
```
Comment (Croydon) [inline docs/reviewing.md:58]: This does simply not work as there are tons of custom licenses out there that do not have an entry on SPDX. So it is more a "must, if possible" situation

Comment (prince-chrismc) [reply docs/reviewing.md:58]: Perfect, that answer my opening questions!

Comment (prince-chrismc) [reply docs/reviewing.md:58]:
```suggestion
The mandatory license attribute of each recipe **should** be a [SPDX license](https://spdx.org/licenses/) [short Identifiers](https://spdx.dev/ids/) when applicable.
```

**Review by uilianries — APPROVED** [review]

**Review by SSE4 — APPROVED** [review]

**Comment by prince-chrismc** [comment]
@Croydon can you please review again, I have another item I'd like to add

**Review by intelligide — APPROVED** [review]

**Pull request merged**